keep/parser/tests_for_parser/resources/providers_for_testing.yaml

```diff
@@ -0,0 +1,3 @@
+slack-demo:
```

Comment:
this could be added to the alert file itself or be mocked in the test itself (which I think is better)

Reply:
The reason behind the  separation between the two  is that  we also want to test the case  where the function is called without the necessary  providers file. It is  because of that test I've discovered the dependency of the different calls to the parse function that is a bug.